doc/getting_started/installation_win.rst

```diff
@@ -17,29 +17,39 @@ Windows system with the latest updates installed.
 
 .. _windows_requirements:
 
-Installing Requirements and Dependencies
-****************************************
+Install Requirements and Dependencies
+*************************************
```

Comment:
You forgot the `NOTE FOR DOCS AUTHORS: DO NOT PUT DOCUMENTATION BUILD DEPENDENCIES HERE.` in the Windows section

Reply:
Whoops, thanks!